src/sass/components/_variables.scss

```
@@ -0,0 +1,8 @@
+// Settings to be used with vuetify base components
+@forward 'vuetify/settings' with (
+
+  // default button SASS variables for vuetify
+  $button-font-size: 16px,
+  $button-font-weight: 600,
+  $button-text-letter-spacing: .1px,
+);
```

There should be a newline at the end of the file, not sure if this suggestion adds it properly:
```suggestion
);

```